app/views/records/batch.html

```diff
@@ -23,7 +23,7 @@
             "name": "batchNumber",
             "fieldset": {
               "legend": {
-                "text": ("Which batch was used?" if vaccination.vaccinationToday == 'yes' else "Which batch was used?"),
+                "text": ("Which batch was it?" if vaccination.vaccinationToday == 'yes' else "Which batch was it?"),
```

Comment:
Looks like we don't need this logic as it's the same text either way? In which case the code can be simpler.

```suggestion
                "text": pageName,
```